pom.xml

```diff
@@ -17,7 +17,7 @@
         <dependency>
             <groupId>net.java.dev.jna</groupId>
             <artifactId>jna</artifactId>
-            <version>5.6.0</version>
+            <version>5.7.0</version>
```

Comment:
This cannot use the newest JNA. There seems to be some API changes and I got a lot of "no-arg constructor" errors when it tries to make CGPoint and I didn't feel like debugging it.